### ISSUE TITLE
Ubuntu apt-key installation instructions pulls in unknown GPG key

### ISSUE THREAD

**Issue by aastaneh** [iovisor/bcc#1983]
https://github.com/iovisor/bcc/blob/master/INSTALL.md#ubuntu---binary references a GPG key ending with D4284CDD, which actually references two keys!

```
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D4284CDD
[sudo] password for amin: 
Executing: /tmp/apt-key-gpghome.hVvH6NnXtF/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys D4284CDD

gpg: key 4052245BD4284CDD: public key "Brenden Blanco <bblanco@plumgrid.com>" imported
gpg: key 27E8E08BD4284CDD: public key "Totally Legit Signing Key <mallory@example.org>" imported
gpg: Total number processed: 2
gpg:               imported: 2
```

A Google search referenced a recent post on SecLists https://seclists.org/oss-sec/2018/q3/174 and a tool that can bruteforce key ids: https://github.com/jwilk/stopgp32

I advise changing the docs to reference the full keyid 4052245BD4284CDD to prevent users from unknowingly installing malicious signing keys into their Apt package manager.

Thanks!

**Comment by pchaigno** [comment]
@aastaneh Thanks for the report! Do you want to submit a pull request for that?

**Comment by aastaneh** [comment]
No need, it seems to have been addressed in e55c68acfad72ce19cc51ecc73d4938bd5a6ff86. Thank you!
